core/crypto.mk

```diff
@@ -81,6 +81,8 @@ CFG_CRYPTO_SHA512_ARM_CE ?= $(CFG_CRYPTO_SHA512)
 CFG_CORE_CRYPTO_SHA512_ACCEL ?= $(CFG_CRYPTO_SHA512_ARM_CE)
 CFG_CRYPTO_SM3_ARM_CE ?= $(CFG_CRYPTO_SM3)
 CFG_CORE_CRYPTO_SM3_ACCEL ?= $(CFG_CRYPTO_SM3_ARM_CE)
+CFG_CRYPTO_SM4_ARM_CE ?= $(CFG_CRYPTO_SM4)
+CFG_CORE_CRYPTO_SM4_ACCEL ?= $(CFG_CRYPTO_SM4_ARM_CE)
```

Comment:
You don't need that, because when `CFG_CRYPTO_WITH_CE82` is enabled then `CFG_CRYPTO_WITH_CE` is necessarily enabled, too.

Reply:
I'm a little bit confused, do you mean `CFG_CRYPTO_SM4_ARM_CE` is unnecessary?

Reply:
No sorry, I mean it should be done in the `CFG_CRYPTO_WITH_CE` block

Reply:
I still think it should be done in `CFG_CRYPTO_WITH_CE82`, since SM4E is an optional Cryptographic Extension for ARMv8.2-A, not for ARMv8-A. For platforms on ARMv8-A (e.g., `HiSilicon D06`) who only enabled `CFG_CRYPTO_WITH_CE`,  `CFG_CRYPTO_SM4_ARM_CE` should not be done.

Reply:
Yes, you're right. I meant "it should be done in the `CFG_CRYPTO_WITH_CE82`" in the above comment., sorry for the mistake. Please move these two lines up.